test/jest.matchers.ts

```diff
@@ -0,0 +1,130 @@
+import { BigNumber, BigNumberish } from "ethers";
+
+expect.extend({
+  toBnEq: (b1: BigNumberish, b2: BigNumberish) => {
```

Comment:
not with jest, but with mocha yep. We use jest just because we are more confortable with jest and it is much used than mocha, but the result is the same.